java/api-test/api-usage-example/src/main/java/InlineFunctionTest.java

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft. All rights reserved.
+import com.azure.ai.openai.OpenAIClientBuilder;
+import com.azure.core.credential.AzureKeyCredential;
+import com.microsoft.openai.AzureOpenAiClient;
+import com.microsoft.openai.OpenAIAsyncClient;
+import com.microsoft.semantickernel.Kernel;
+import com.microsoft.semantickernel.KernelConfig;
+import com.microsoft.semantickernel.builders.SKBuilders;
+import com.microsoft.semantickernel.textcompletion.CompletionFunctionDefinition;
+import com.microsoft.semantickernel.textcompletion.CompletionSKContext;
+import com.microsoft.semantickernel.textcompletion.CompletionSKFunction;
+import com.microsoft.semantickernel.textcompletion.TextCompletion;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Properties;
+
+public class InlineFunctionTest {
+    public static final String AZURE_CONF_PROPERTIES = "conf.properties";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(InlineFunctionTest.class);
+
+    private static final String MODEL = "text-davinci-003";
+
+    public static String getToken(String configName) throws IOException {
+        return getConfigValue(configName, "token");
+    }
+
+    public static String getEndpoint(String configName) throws IOException {
+        return getConfigValue(configName, "endpoint");
+    }
+
+    private static String getConfigValue(String configName, String propertyName)
+            throws IOException {
+        String home = new File(System.getProperty("user.home")).getAbsolutePath();
```

Comment:
Is the idea of using `user.home` to keep the config information out of the workspace and avoid accidental check-ins?